src/js/gi/reducers/index.js

```diff
@@ -9,7 +11,7 @@ import InstitutionDetailsFilterReducer from './filter';
 import ProfileReducer from './profile';
 import CalculatorReducer from './calculator';
 
-const rootReducer = {
+const rootReducer = combineReducers({
```

Comment:
Oh, I see now that the common store is doing the `combineReducers` already. So it looks like I'll have to do this within the tests themselves.